blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -57,7 +57,23 @@ export const actionHandlers = {
         return state.set('currentRuns', null);
     },
     [ACTION_TYPES.SET_CURRENT_RUN_DATA](state, { payload }): State {
-        return state.set('currentRuns', payload);
+        return state.set('currentRuns', payload.map((run) => {
+            if (run._class === 'io.jenkins.blueocean.service.embedded.rest.QueueItemImpl') {
```

Comment:
Right, it looks like there is a way to look up the capabilities api without doing a round trip - @cliffmeyers has done this. This means that instead of looking directly at the impl class, looking up the capabilitist list and matching against that would replace the string comparison (but do the same thing, functionally). 

May have to punt that to another ticket as help is needed and Cliff not around this week.